### PR TITLE
Fix Dockerfile and dev requirements

### PR DESCRIPTION
**Pull request by danrue**
Signed-off-by: Dan Rue <dan.rue@linaro.org>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -5,8 +5,9 @@ RUN apt-get update && \
     python3-dateutil \
     python3-django \
     python3-celery \
-    python3-djano-celery \
+    python3-django-celery \
     python3-whitenoise \
+    python3-pip \
```
Comment (terceiro) [inline Dockerfile]: this should not be needed since it's using everything from Debian already

File: Dockerfile [hunk 2]
```diff
@@ -24,6 +25,8 @@ RUN env
 RUN python3 -m squad.frontend
 RUN ./manage.py collectstatic --noinput
 
+RUN chown -R www-data /app
```
Comment (terceiro) [inline Dockerfile]: why do you need this? having source tree be read-only for the user that runs the service is better for security

Comment (danrue) [reply Dockerfile]: It's because I was using it as a dev environment, but there's probably a better way. I'll update the PR

**Comment by danrue** [comment]
Closing in favor of #95

**Pull request closed without merging**